projects/vdk-core/tests/functional/run/test_run_errors.py

```diff
@@ -33,7 +37,7 @@ def tmp_termination_msg_file(tmpdir) -> pathlib.Path:
 
 
 def test_initialize_step_user_error(tmp_termination_msg_file):
-    errors.clear_intermediate_errors()
+    errors.resolvable_context().clear()
```

Comment:
I think this needs to be done at the start of absolutely every test. 
And not just tests in this file.